src/sentry/static/sentry/app/api.tsx

```diff
@@ -299,13 +299,25 @@ export class Client {
    */
   request(path: string, options: Readonly<RequestOptions> = {}): Request {
     const method = options.method || (options.data ? 'POST' : 'GET');
+
+    let fullUrl = buildRequestUrl(this.baseUrl, path, options.query);
+
     let data = options.data;
 
     if (!isUndefined(data) && method !== 'GET') {
       data = JSON.stringify(data);
     }
 
-    const fullUrl = buildRequestUrl(this.baseUrl, path, options.query);
+    // TODO(epurkhiser): Mimicking the old jQuery API, data could be a string /
+    // object for GET requets. jQuery just sticks it onto the URL as query
+    // parameters
+    if (method === 'GET' && data) {
+      const paramData = typeof data === 'string' ? data : jQuery.param(data);
+
+      if (paramData.length > 0) {
+        fullUrl = fullUrl + (fullUrl.indexOf('?') !== -1 ? '&' : '?') + data;
```

Comment:
@EvanPurkhiser I feel like we should either have a utility function that does this or add it if it doesn't exist

Reply:
IMO I'm fine with this since it's trivial, and that's the kind of utility function no one is going to look for